client/views/helpers/qr_code/qr_code.scss

```diff
@@ -0,0 +1,36 @@
+.modaltopmessage > h2 {
+  line-height: 25px;
+  margin-top: 5px;
+  color: black;
+}
+
+.qr_codeouter {
```

Comment:
@singhsanket143 can you fix the code style of this file?

Reply:
@peter-hank Updated the file